12.0-buster/base_requirements.txt

```diff
@@ -40,7 +40,8 @@ python-dateutil==2.5.3
 pytz==2016.7
 pyusb==1.0.0
 qrcode==5.3
-reportlab==3.3.0
+feedparser==6.0.8
```

Comment:
Why re-introduced `feedparser` here? It has been removed on purpose in a previous release: a9bcc9f09432902ed36a370bc5e4f993c7fc983f